docs/community/styleguide/formatting.md

```diff
@@ -0,0 +1,69 @@
+---
```

Comment:
Amazing work so far! Your document content is well-structured, nice job!  ✨ I have some suggestions to improve the clarity and usefulness of your guide:

----

### Documentation formatting:

1. It seems like you have a typo here: `"Use the following Use the following syntax to apply a style."` Please fix it.

### Code Blocks:

1. The indentation rule can be improved by **specifying the number of spaces that should be used for indentation**. As it stands, it is vague and leaves it open to individual interpretation.

2. The point "For yaml document, enclose with triple backticks and specify `yaml`." could be clearer by mentioning that this is for showing code blocks of YAML content. That sentence also has a typo, it is missing an article before the word `yaml`, please fix it. 

### Spacing:

1. In the spacing section, **dictate detailed guidelines for achieving appropriate line spacing and consistent indentation in Markdown**. (one example isn't enough) For example, **how many lines should be left between sections**, **should indentation be 2 or 4 spaces**, etc.